examples/gce-acme/main.tf

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "acme_email" {}
```

Comment:
Usually I like to put variables in their own `variables.tf` file.

Reply:
For the examples I've been just keeping them in the main.tf file, because usually it's just 1 or 2 variables, but ya I can move them to a variables.tf file since there are several here.